### PR TITLE
🧹 Change compiler version for USDC contracts (Reverts #129)

### PR DESCRIPTION
**Pull request by ravinagill15**
This reverts commit [65c78b5ffefc683f5a6167a1166760d5f4aa0cbf](https://github.com/stargate-protocol/stargate-v2/pull/129). 

The reverted commit allowed us to deploy USDC with just the raw bytecode, which is not an ideal approach. Since then, we have decided on a better approach -- we will always deploy USDC from [circle's repo](https://github.com/circlefin/stablecoin-evm) and configure the address of the USDC Proxy within stargate [(see here)](https://github.com/stargate-protocol/stargate-v2/pull/176/files#diff-ba4479457e246e8302ee62e180e60373999e1673da83a5715b046d0ef8d45092R443)

### CODE REVIEW

File: packages/stg-evm-v2/hardhat.config.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ const networks: NetworksUserConfig = {
     //
     'arbsep-testnet': {
         eid: EndpointId.ARBSEP_V2_TESTNET,
-        url: process.env.RPC_URL_ARBITRUM_TESTNET || 'https://arbitrum-sepolia.gateway.tenderly.co',
+        url: process.env.RPC_URL_ARBITRUM_TESTNET || 'https://arbitrum-sepolia.blockpi.network/v1/rpc/public',
```
Comment (janjakubnanista) [inline packages/stg-evm-v2/hardhat.config.ts:121]: Are these changes intentional?

**Review by janjakubnanista — APPROVED** [review]

File: packages/stg-evm-v2/hardhat.config.ts (hunk 1 above)
Comment (ravinagill15) [reply packages/stg-evm-v2/hardhat.config.ts:121]: Yeah these were just done to make the testnet build pass -- the previous RPCs were causing it to fail
